### PR TITLE
Liveradio e2e test live

### LINKED ISSUE #3996: Live radio Cypress tests need to be environment agnostic
**Is your feature request related to a problem? Please describe.**
[This](https://github.com/bbc/simorgh/blob/69fdb3d8c451892aad99bf43a3011ea968d02eb5/cypress/integration/pages/liveRadio/testsForAMPOnly.js#L32) and [this](https://github.com/bbc/simorgh/blob/69fdb3d8c451892aad99bf43a3011ea968d02eb5/cypress/integration/pages/liveRadio/testsForCanonicalOnly.js#L22) need to be environment agnostic and not hardcoded to `www.test.bbc.com`. 

James D suggested trying something like `iframe[src$=/ws/av-embeds/media/...]` - https://github.com/bbc/simorgh/pull/3990#discussion_r328733741

**Describe the solution you'd like**
The test must be environment agnostic.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- ~[ ] This feature is expected to need manual testing.~

**Additional context**

### LINKED ISSUE #3997: Add Live radio cypress live environment tests
**Is your feature request related to a problem? Please describe.**
The LIVE cypress e2e tests should be completed after or in the same PR as https://github.com/bbc/simorgh/issues/3996

**Describe the solution you'd like**
Add LIVE cypress e2e test for the 5 live radio pages.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Cypress test will be sufficient 

- ~[ ] This feature is expected to need manual testing.~

**Additional context**

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #3996
Resolves #3997 

**Overall change:** adds live radio e2e's for live env and make test env agnostic

### CODE REVIEW

File: cypress/integration/pages/liveRadio/testsForAMPOnly.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
             const { id, externalId } = body.content.blocks[2];
             const serviceId = getMappedServiceId(externalId);
             cy.get(
-              `amp-iframe[src="${`https://www.test.bbc.com/ws/av-embeds/media/${serviceId}/${id}/amp`}"]`,
+              `amp-iframe[src="${envConfig.baseUrl}/${serviceId}/${id}/amp"]`,
```
Comment (dr3) [inline cypress/integration/pages/liveRadio/testsForAMPOnly.js]: Think this needs `/ws/av-embeds/media/` still

File: cypress/support/config/services.js [hunk 2]
```diff
@@ -23,10 +23,7 @@ const services = {
         smoke: false,
       },
       liveRadio: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/afaanoromoo/bbc_afaanoromoo_radio/liveradio',
+        path: '/afaanoromoo/bbc_afaanoromoo_radio/liveradio',
         smoke: false,
```
Comment (dr3) [inline cypress/support/config/services.js:27]:
```suggestion
        smoke: true,
```

**Review by dr3 — CHANGES_REQUESTED** [review]

File: cypress/support/config/services.js [hunk 3]
```diff
@@ -1311,10 +1299,7 @@ const services = {
     variant: 'default',
     pageTypes: {
       articles: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/tigrinya/articles/c12g32eldk6o',
+        path: '/tigrinya/articles/c12g32eldk6o',
```
Comment (amywalkerdev) [inline cypress/support/config/services.js]: article? Wrong page type.

File: cypress/support/config/services.js [hunk 4]
```diff
@@ -103,11 +100,8 @@ const services = {
         smoke: false,
       },
       liveRadio: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/amharic/bbc_amharic_radio/liveradio',
-        smoke: false,
+        path: '/amharic/bbc_amharic_radio/liveradio',
+        smoke: true,
```
Comment (dr3) [inline cypress/support/config/services.js:104]: Only 1 of these should be smoke :(

**Review by dr3 — CHANGES_REQUESTED** [review]

File: cypress/support/config/services.js [hunk 5]
```diff
@@ -23,11 +23,8 @@ const services = {
         smoke: false,
       },
       liveRadio: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/afaanoromoo/bbc_afaanoromoo_radio/liveradio',
-        smoke: false,
+        path: '/afaanoromoo/bbc_afaanoromoo_radio/liveradio',
+        smoke: true,
```
Comment (pjlee11) [inline cypress/support/config/services.js]:
```suggestion
        smoke: false,
```

File: cypress/support/config/services.js [hunk 6]
```diff
@@ -514,11 +508,8 @@ const services = {
         smoke: false,
       },
       liveRadio: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/indonesia/bbc_indonesian_radio/liveradio',
-        smoke: false,
+        path: '/indonesia/bbc_indonesian_radio/liveradio',
+        smoke: true,
```
Comment (pjlee11) [inline cypress/support/config/services.js]:
```suggestion
        smoke: false,
```

File: cypress/support/config/services.js [hunk 7]
```diff
@@ -586,11 +577,8 @@ const services = {
         smoke: false,
       },
       liveRadio: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/korean/bbc_korean_radio/liveradio',
-        smoke: false,
+        path: '/korean/bbc_korean_radio/liveradio',
+        smoke: true,
```
Comment (pjlee11) [inline cypress/support/config/services.js]:
```suggestion
        smoke: false,
```

File: cypress/support/config/services.js [hunk 8]
```diff
@@ -1329,11 +1317,8 @@ const services = {
         smoke: false,
       },
       liveRadio: {
-        path:
-          Cypress.env('APP_ENV') === 'live'
-            ? undefined
-            : '/tigrinya/bbc_tigrinya_radio/liveradio',
-        smoke: false,
+        path: '/tigrinya/bbc_tigrinya_radio/liveradio',
+        smoke: true,
```
Comment (pjlee11) [inline cypress/support/config/services.js]:
```suggestion
        smoke: false,
```

File: cypress/integration/pages/liveRadio/testsForAMPOnly.js [hunk 9]
```diff
@@ -31,7 +31,7 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
             const { id, externalId } = body.content.blocks[2];
             const serviceId = getMappedServiceId(externalId);
             cy.get(
-              `amp-iframe[src="${`https://www.test.bbc.com/ws/av-embeds/media/${serviceId}/${id}/amp`}"]`,
+              `amp-iframe[src="${envConfig.baseUrl}/ws/av-embeds/media/${serviceId}/${id}/amp"]`,
```
Comment (dr3) [inline cypress/integration/pages/liveRadio/testsForAMPOnly.js]: Will this fail locally now?

Comment (dr3) [reply cypress/integration/pages/liveRadio/testsForAMPOnly.js]:
```
1) liveRadio - afaanoromoo - Canonical Canonical Tests for afaanoromoo liveRadio live radio body should render an audio player embed:
CypressError: Timed out retrying: Expected to find element: 'iframe[src="http://localhost.bbc.com:7080/ws/av-embeds/media/bbc_afaanoromoo_radio/liveradio"]', but never found it.[0m[90m
```

Comment (pjlee11) [reply cypress/integration/pages/liveRadio/testsForAMPOnly.js]: Probably, checking

Comment (dr3) [reply cypress/integration/pages/liveRadio/testsForAMPOnly.js]: Can add `liveRadioIframeBaseUrl` to `simorgh/cypress/support/config/envs.js`

**Review by dr3 — APPROVED** [review]
LGTM assuming passes

**Review by chris-hinds — APPROVED** [review]

**Comment by pjlee11** [comment]
On local dev passes against CYPRESS_ENV `local`. Trying `test` and `live` now

**Pull request merged**